src/sentry/web/urls.py

```diff
@@ -146,6 +148,16 @@
         ErrorPageEmbedView.as_view(),
         name="sentry-error-page-embed",
     ),
+    re_path(
+        r"^toolbar/iframe$",
```

Comment:
Lets also nest this under `/organizations/<org_slug>`

and we should followup and make sure we're writing it the correct way so when subdomains are enabled (like on sentry.io) things are still working, or at least not auto-redirecting into the subdomain.

Reply:
:+1: for having it organization scoped.